<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QMainWindow" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="pagesWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="stackedWidget_Page1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ProfilesPage" name="profilesPage" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="stackedWidget_Page2">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="GeneralPage" name="generalPage" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="stackedWidget_Page3">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="programsPage" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="stackedWidget_Page4">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="UpdatePage" name="updatePage" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actShowProfilesPage"/>
   <addaction name="actShowGeneralPage"/>
   <addaction name="actShowProgramsPage"/>
   <addaction name="actShowUpdatePage"/>
  </widget>
  <action name="actShowProfilesPage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string> Audio profiles</string>
   </property>
  </action>
  <action name="actShowGeneralPage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>General</string>
   </property>
  </action>
  <action name="actShowProgramsPage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Programs</string>
   </property>
  </action>
  <action name="actShowUpdatePage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProfilesPage</class>
   <extends>QWidget</extends>
   <header location="global">profilespage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GeneralPage</class>
   <extends>QWidget</extends>
   <header>generalpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UpdatePage</class>
   <extends>QWidget</extends>
   <header>updatepage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
